pkg/volume/csi/csi_attacher.go

```diff
@@ -154,7 +155,15 @@ func (c *csiAttacher) waitForVolumeAttachment(volumeHandle, attachID string, tim
 
 func (c *csiAttacher) waitForVolumeAttachmentInternal(volumeHandle, attachID string, timer *time.Timer, timeout time.Duration) (string, error) {
 	glog.V(4).Info(log("probing VolumeAttachment [id=%v]", attachID))
-	attach, err := c.k8s.StorageV1beta1().VolumeAttachments().Get(attachID, meta.GetOptions{})
+	var attach *storage.VolumeAttachment
+	var err error
+	err = retry.RetryOnConflict(retry.DefaultRetry, func() error {
```

Comment:
`RetryOnConflict()` retries given call when the call returned `metav1.StatusReasonConflict`. There is no way how GET can return this error. This code therefore probably does not do anything.